ensime-search.el

```diff
@@ -285,7 +285,7 @@
 	(goto-char target-point)
 	(setq ensime-search-selection-overlay
 	      (ensime-make-overlay target-point (point-at-eol)
-				   nil 'ensime-warnline-highlight))
+				   nil '(:face 'ensime-warnline-highlight)))
```

Comment:
Oups... Didn't know that! Sorry!
I'm new to elisp, but I will try to improve fast ;-)
After all, I am really impressed by the capabilities of Emacs and Ensime!!
Elisp is on the best way in getting my second favorite language after Scala....!